test/Transform/ATenVisualGraph/tiny_yolo_v2_block.mlir

```diff
@@ -10,7 +10,23 @@
 
 // RUN: aten-opt %s -aten-visual-graph='operators-supported-path=%S/../../../lib/Transform/operators_supported.json' | FileCheck %s
 // CHECK-LABEL:     "{{.*}}": {
-//
+// CHECK-LABEL:     "connections"
+// CHECK-LABEL:     {
```

Comment:
the use of https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-label-directive everywhere looks odd and that you really just want `CHECK:`. Could you check?